src/plugins/data/common/query/timefilter/get_time.ts

```diff
@@ -34,6 +34,18 @@ export function calculateBounds(
   };
 }
 
+export function getAbsoluteTimeRange(
```

Comment:
Why do we need both this function and `calculateBounds`?

Reply:
`calculateBounds` returns slightly different format from `getAbsoluteTimeRange`. 
now well aware of `calculateBounds` usages and use cases so seemed fine to make a similar function that returns similar but different thing nearby.

Reply:
couldn't we reuse calculateBounds like this:
```
 const { min, max } = calculateBounds(timeRange, { forceNow });
 return { from: min ? min.toISOString() : timeRange.from }, to: max ? max.toISOString() : timeRange.to }
```

Reply:
Improved 👍